### PR TITLE
Added search-v2-operator-bundle component to prow

### PR DESCRIPTION
**Pull request by dislbenn**
Signed-off-by: dislbenn <dbennett@redhat.com>

### CODE REVIEW

**Review by xrajesh — APPROVED** [review]

File: ci-operator/config/stolostron/search-v2-operator-bundle/stolostron-search-v2-operator-bundle-main.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- as: git-fast-forward
+  postsubmit: true
+  steps:
+    env:
+      DESTINATION_BRANCH: release-2.7
+    workflow: ocm-ci-fastforward
+zz_generated_metadata:
+  branch: main
+  org: stolostron
+  repo: search-v2-operator-bundle
```
Comment (xrajesh) [inline ci-operator/config/stolostron/search-v2-operator-bundle/stolostron-search-v2-operator-bundle-main.yaml:45]: @dislbenn - I see some 2.5 and 2.6 failures , we care only about 2.7 and main configurations - We can leave the 2.5 and 2.6 prow jobs as is . If that helps .

**Comment by dislbenn** [comment]
Closing this PR.

**Pull request closed without merging**